designer/server/src/test/scala/pl/touk/nussknacker/ui/api/DeploymentApiHttpServiceBusinessSpec.scala

```diff
@@ -75,16 +74,24 @@ class DeploymentApiHttpServiceBusinessSpec
 
   private val scenarioName = "batch-test"
 
+  private val sourceNodeId = "fooSourceNodeId"
+
   private val scenario = ScenarioBuilder
     .streaming(scenarioName)
-    .source("source", "table", "Table" -> Expression.spel("'transactions'"))
+    .source(sourceNodeId, "table", "Table" -> Expression.spel("'transactions'"))
     .emptySink(
       "sink",
       "table",
       "Table" -> Expression.spel("'transactions_summary'"),
       "Value" -> Expression.spel("#input")
     )
 
+  private val correctDeploymentRequest = s"""{
```

Comment:
(Not related to this line) When I tried to test this locally using batch dev setup, I get a "Comment is required." message. So to test manually this I have to disable the `deploymentCommentSettings` globally. Or is there another way / other way to pass the comment?

Reply:
I'll check it

Reply:
I added the new request parameter: `comment`. I needed to extract another test class because designer config is setup on class level. Now it should work also on the staging